notNeededPackages.json

```
@@ -3432,12 +3432,102 @@
+            "asOfVersion": "5.2.0"
+        },
+        {
+            "libraryName": "@storybook/addon-backgrounds",
+            "typingsPackageName": "storybook__addon-backgrounds",
+            "sourceRepoURL": "https://github.com/storybooks/storybook",
+            "asOfVersion": "5.2.0"
+        },
+        {
+            "libraryName": "@storybook/addon-centered",
+            "typingsPackageName": "storybook__addon-centered",
+            "sourceRepoURL": "https://github.com/storybooks/storybook",
+            "asOfVersion": "5.2.0"
+        },
+        {
+            "libraryName": "@storybook/addon-jest",
+            "typingsPackageName": "storybook__addon-jest",
+            "sourceRepoURL": "https://github.com/storybooks/storybook",
+            "asOfVersion": "5.2.0"
+        },
+        {
+            "libraryName": "@storybook/addon-knobs",
+            "typingsPackageName": "storybook__addon-knobs",
+            "sourceRepoURL": "https://github.com/storybooks/storybook",
+            "asOfVersion": "5.2.0"
+        },
+        {
+            "libraryName": "@storybook/addon-links",
+            "typingsPackageName": "storybook__addon-links",
+            "sourceRepoURL": "https://github.com/storybooks/storybook",
+            "asOfVersion": "5.2.0"
+        },
         {
             "libraryName": "@storybook/addon-notes",
             "typingsPackageName": "storybook__addon-notes",
             "sourceRepoURL": "https://github.com/storybooks/storybook",
             "asOfVersion": "5.0.0"
         },
+        {
+            "libraryName": "@storybook/addon-options",
+            "typingsPackageName": "storybook__addon-options",
+            "sourceRepoURL": "https://github.com/storybooks/storybook",
+            "asOfVersion": "5.2.0"
+        },
+        {
+            "libraryName": "@storybook/addon-viewport",
+            "typingsPackageName": "storybook__addon-viewport",
+            "sourceRepoURL": "https://github.com/storybooks/storybook",
+            "asOfVersion": "5.2.0"
+        },
+        {
+            "libraryName": "@storybook/addons",
+            "typingsPackageName": "storybook__addons",
+            "sourceRepoURL": "https://github.com/storybooks/storybook",
+            "asOfVersion": "5.2.0"
+        },
+        {
+            "libraryName": "@storybook/channels",
+            "typingsPackageName": "storybook__channels",
+            "sourceRepoURL": "https://github.com/storybooks/storybook",
+            "asOfVersion": "5.2.0"
+        },
+        {
+            "libraryName": "@storybook/html",
+            "typingsPackageName": "storybook__html",
+            "sourceRepoURL": "https://github.com/storybooks/storybook",
+            "asOfVersion": "5.2.0"
+        },
+        {
+            "libraryName": "@storybook/preact",
+            "typingsPackageName": "storybook__preact",
+            "sourceRepoURL": "https://github.com/storybooks/storybook",
+            "asOfVersion": "5.2.1"
```

I’d prefer if this was:
```suggestion
            "asOfVersion": "5.2.0"
```
But I don’t know how that would affect `types‑publisher`.

---

Like I said in my PR description:
> Set to `5.2.1` since `5.2` typings were already published here even tho that package version wasn't released yet